src/cluster_legacy.c

```diff
@@ -116,6 +116,14 @@ int verifyClusterNodeId(const char *name, int length);
 sds clusterEncodeOpenSlotsAuxField(int rdbflags);
 int clusterDecodeOpenSlotsAuxField(int rdbflags, sds s);
 
+/* Only primary that own slots have the voting rights.
+ * And only voting primary will be counted in cluster size.
+ *
+ * Returns 1 if the node has voting rights, otherwise returns 0. */
```

Comment:
Nit - the cluster size comment is out of place since we don't compute the cluster size in this function.
```suggestion
/* Only primaries that own slots have voting rights.
 * Returns 1 if the node has voting rights, otherwise returns 0. */
```

Reply:
right, took it, thanks.